framework/src/service/ServiceListeners.cpp

```diff
@@ -24,6 +24,8 @@
 
 US_MSVC_PUSH_DISABLE_WARNING(4180) // qualifier applied to function type has no meaning; ignored
 
+#include <cassert>
```

Comment:
I did use asserts in the interim but not anymore. Didn't remove the needless #include though.